.pylintrc

```diff
@@ -252,7 +252,7 @@ ignore-docstrings=yes
 ignore-imports=yes
 
 # Minimum lines number of a similarity.
-min-similarity-lines=12
+min-similarity-lines=4
```

Comment:
Please leave this at 12. We will need to fix the libs as well to return them to 12. Typing adds similar lines to library files, and we ran into this issue during the Sprints, where 4 was no longer sufficient. I do not want to disable the duplicate code check entirely, but returning it to 12 is fine.